samples/ComponentExamples/src/examples/RenderSingleVideoTileExample.tsx

```diff
@@ -69,12 +69,14 @@ export const RenderVideoTileExample = (): JSX.Element => {
     <>
       <h3> Render single video stream in your own component </h3>
       <span>Click camera button to see what happens:</span>
-      <CameraButton {...cameraButtonProps} />
+      <CameraButton {...cameraButtonProps} id="camera-button" />
```

Comment:
I noticed IdentifierProvider marked all data-ui-id string to required - which doesn't work very well in this case(we only need to override one component id here)

And also we are exporting it as _IdentifierProvider, anyone knows why we add a _ prefix here?

Reply:
It is an `@internal` symbol exported in the API for automation testing purposes.

All `@internal` symbols must start with `_`, per api-extractor. It's `@internal` because we don't make API stability guarantees for it.

@JamesBurnside  @anjulgarg I know I initially requested `_identifierProvider`, but we only use it in `@internal/react-components` and not `@internal/react-composites`. Since we've agreed that generated DOM is not part of our public API, would we be OK removing this entirely and just using the `data-ui-id` attributes directly everywhere?

Reply:
If we haven't decided whether to expose it or not, maybe we could keep it internally - we don't see a need from customer to do automation test on top of our UI SDK for now - maybe it could happen in the future? Then it is the time to improve this provider. For now, I will keep what I have as implementation of tests

Reply:
I'm +1 removing it and just having hard-coded data-ui-ids